addons/payment/views/payment_templates.xml

```diff
@@ -244,14 +244,19 @@
                             <!-- === Token name === -->
                             <span class="payment_option_name" t-esc="token.name"/>
                             <!-- === "V" check mark === -->
-                            <i t-if="token.verified" class="fa fa-check text-success"
-                               title="This payment method has been verified by our system."
-                               role="img"
-                               aria-label="Ok"/>
-                            <i t-else="" class="fa fa-check text-muted"
-                               title="This payment method has not been verified by our system."
-                               role="img"
-                               aria-label="Not verified"/>
+                            <t t-if="token.acquirer_id.name=='Test'">
+                                <span class="badge badge-success">Test Token</span>
```

Comment:
The part should be done in an xpath from the payment_test acquirer module. In practice, this will always work because payment_test is auto_install, but still... Let's do things right! :)